lib/versioning/rez/readme.md

```diff
@@ -0,0 +1,16 @@
+rez versioning was developed to support the [rez Package Manager](https://github.com/nerdvegas/rez).
+It's based on Semantic versioning but includes its own concept of ranges.
```

Comment:
```suggestion
It's based on Semantic Versioning but includes its own concept of ranges.
```

If `rez` is based on the Semantic Versioning spec (with the exception of different range behavior). I think we should capitalize the whole phrase "Semantic Versioning", as that's how it's styled in the Semantic Versioning specification as found here: https://semver.org/